### PR TITLE
Fixing some phpstorm findings in tests

### PR DESCRIPTION
**Pull request by franmomu**

### CODE REVIEW

File: tests/Action/AppendFormFieldElementActionTest.php [hunk 1]
```diff
@@ -22,6 +22,7 @@
 use Sonata\AdminBundle\Admin\FieldDescriptionInterface;
 use Sonata\AdminBundle\Admin\Pool;
 use Sonata\AdminBundle\Model\ModelManagerInterface;
+use stdClass;
```
Comment (OskarStark) [inline tests/Action/AppendFormFieldElementActionTest.php]: Why this change? I mean this should only be fixed by a cs rule.

Comment (core23) [inline tests/Action/AppendFormFieldElementActionTest.php]: AFAIK we don't use import for internal classes

Comment (franmomu) [reply tests/Action/AppendFormFieldElementActionTest.php]: Ok, I saw it in https://github.com/sonata-project/SonataSeoBundle/pull/358/files#diff-dbdd1855f86b07a0c76f12457036ab73R19, that's why I thought it was ok

**Comment by core23** [comment]
Thanks @franmomu
